extensions/wikia/Flags/scripts/FlagsModal.js

```diff
@@ -88,7 +88,10 @@ require(
 					type: loader.MULTI,
 					resources: {
 						mustache: '/extensions/wikia/Flags/controllers/templates/FlagsController_editForm.mustache,/extensions/wikia/Flags/controllers/templates/FlagsController_editFormEmpty.mustache,/extensions/wikia/Flags/controllers/templates/FlagsController_editFormException.mustache',
-						styles: '/extensions/wikia/Flags/styles/EditFormModal.scss'
+						styles: '/extensions/wikia/Flags/styles/EditFormModal.scss',
+						params: {
+							sassParams: window.wgSassParams // Ensure per-theme colors Varnish cache and mcache
+						}
```

Comment:
I'd rather pass `sassParams` in `loader` module when SASS assets are requested instead of every place when the loader is used to fetch SASS styles.